tests/http-status.js

```diff
@@ -1,192 +1,108 @@
-
-  if (fails.internal.size > 0) {
-    statusStr = chalk.red(`[FAIL]`);
-    exitCode = 1;
-  }
-  else if (fails.external.size > 0) {
-    statusStr = chalk.yellow(`[WARN]`);
-    githubCommentLines = lines;
-  }
-
-  const noInternalLinks = Object.keys(foundLinks).length <= 1;
-  if (noInternalLinks) {
-    lines.push(`${chalk.red(`[FAIL]`)} Only one page was tested, so the main page has no internal links.`);
-    exitCode = 1;
-  }
-
-  await createOrUpdatePrComment();
-
-  console.log(statusStr, lines.join(`\n`));
-  process.exit(exitCode);
-
-
-  /**
-   * @returns {Promise.<*>} Promise that is resolved when the comment is created / updated.
-   */
-  async function createOrUpdatePrComment() {
-    try {
-      await pullRequest.checkEnv();
-    }
-    catch (err) {
-      console.error(chalk.yellow(`[WARN]`), `Can't create/delete GitHub comment:`, err); // PR env variables not set
-      process.exit(0);
+    await onServerClosed; // wait until server has been stopped and stdout/stderr were written to console
+    console.log();
+
+    let statusStr = chalk.greenBright(`[PASS]`);
+    let exitCode = 0;
+    let periodOrColon = `.`;
+    if (failingLinks.length > 0) {
+      statusStr = chalk.redBright(`[FAIL]`);
+      exitCode = 1;
+      periodOrColon = `:`;
     }
+    console.log(statusStr, `${failingLinks.length} of ${passingLinks.length + failingLinks.length} tested internal links failed${periodOrColon}`);
+    failingLinks.forEach(link => console.log(`- ${link}`));
+    console.log();
 
-    try {
-      await pullRequest.init();
-      await pullRequest.updateComment({
-        filename: path.relative(path.join(__dirname, `../`), __filename),
-        name: `Broken links`,
-        lines: githubCommentLines
-      });
-    }
-    catch (error) {
-      console.error(`Creating / updating the GitHub PR comment failed.`, error);
-      process.exit(1);
-    }
+    const testTime = new Date() - testStartTime;
+    console.log(chalk.greenBright.bold(`Test took ${testTime / 1000}s.`));
+    process.exit(exitCode);
   }
-});
-console.log(`Started server with process id ${serverProcess.pid}.`);
-
-
-// start checking links after first stdout data from server
-serverProcess.stdout.on(`data`, startLinkChecker);
-function startLinkChecker(data) {
-  serverProcess.stdout.removeListener(`data`, startLinkChecker);
-
-  console.log(`${data}\nStarting HTTP requests ...\n`);
-
-  startTime = new Date();
-  siteChecker.enqueue(BASE_URL);
+  catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+})();
+
+
+/**
+ * Starts the production OFL server by creating a new child process.
+ * Also logs the server's stdout/stderr to console when it stops.
+ * Make sure that 'make nuxt-build' has been called before this function is executed!
+ *
+ * @returns {Promise} Promise that resolves as soon as the OFL server is up and running, or rejects if the process stops with an error before.
+ */
+async function startServer() {
```

Comment:
The exact same function is also declared in `tests/external-links.js`. Can't you reuse it from there (or vice-versa)?